apidoc/Map.yml

```diff
@@ -360,6 +360,7 @@ examples:
             });
             win.open();
 
+examples:
```

Comment:
That produces two "examples" sections (along with [line 315](https://github.com/jawa9000/ti.map/blob/master/apidoc/Map.yml#L315)).